src/actions/LogActions.js

```diff
@@ -35,16 +35,24 @@ export const showSendLogsModal = () => async (dispatch: Dispatch, getState: GetS
 }
 
 const prepareLogs = (text: string) => async (dispatch: Dispatch, getState: GetState) => {
+  const id = Math.floor(Math.random() * 16777215).toString(16)
```

Comment:
Can you use '0xFFFFFF' instead of decimal? It looks less "magic" in hex ;)

Reply:
okaaaay

Reply:
Great. now VScode thinks it's a color and colorizes the text 😏